src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculetion {
+
+    private double baseB;
+    private double baseA;
+    private double height;
+
+    public IsoscelesTrapezoid(
+            double height,
+            double baseA,
+            double baseB,
+            String color,

```

Comment:
The constructor should not take `String name` as a parameter since it's not being used. If you need to set a name for a figure, consider setting it inside the constructor without passing it as a parameter, or remove it if it's not needed.